modules/rosa-scaling-worker-nodes.adoc

```diff
@@ -0,0 +1,57 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-managing-worker-nodes.adoc
+
+[id="rosa-scaling-worker-nodes_{context}"]
+= Scaling worker nodes
+
+[role="_abstract"]
+Worker nodes can be scaled manually if you do not want to configure node autoscaling.
+
+. Procedure
+
+. To find the ID of the machine pool you want to scale and verify the current number of replicas, enter the following command to get a list of the machine pools in a cluster. Each cluster has a default machine pool that is created when you create a cluster.
```

Comment:
It's hard to see what's going on in this step, as there appear to be several potential goals or actions: find the ID, verify the number of replicas, get a list of machine pools. Can you simplify for clarity? Perhaps something like:

> You can find the ID and verify the current number of replicas of the machine pool. To get a list of the default machine pools that were created when you created a cluster, enter the following command:

(Not sure if that retains accuracy.)

Reply:
It doesn't make it a procedure anymore. But I grok you, wrestled with that myself. I've changed it into two steps now. Thanks.